### ISSUE TITLE
th-lift-0.7.5 incompatible with template-haskell-2.11

### ISSUE THREAD

**Issue by hvr** [mboes/th-lift#23]
See also https://ghc.haskell.org/trac/ghc/wiki/Migration/8.0

```
[1 of 1] Compiling Language.Haskell.TH.Lift ( src/Language/Haskell/TH/Lift.hs, dist/dist-sandbox-460590e2/build/Language/Haskell/TH/Lift.o )

src/Language/Haskell/TH/Lift.hs:172:13: error:
    • The constructor ‘DataD’ should have 6 arguments, but has been given 5
    • In the pattern: DataD dcx n vsk cons _
      In the pattern: TyConI (DataD dcx n vsk cons _)
      In a case alternative:
          TyConI (DataD dcx n vsk cons _)
            -> f dcx n (map unTyVarBndr vsk) cons
xcabal: Error: some packages failed to install:
th-lift-0.7.5 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by mboes** [comment]
Thanks for the heads up. Will `th-lift` even be necessary post GHC 8.0. Or does -XDeriveLift effectively make it obsolete? If so I would view th-lift as becoming merely a compat package.

**Comment by hvr** [comment]
Tbh, this was mostly to let you know indirectly that I had to edit in a tighter upper bound at

http://hackage.haskell.org/package/th-lift-0.7.5/revisions/

:-)

I haven't been following the `-XDeriveLift` feature...

**Comment by mboes** [comment]
Fixed by #24.
